scripts/sct_check_dependencies.py

```diff
@@ -214,61 +215,49 @@ def main():
     #         print_ok()
     #     else:
     #         print_warning()
-    #         print '  Detected version: '+version+'. Required version: '+dipy_version
+    #         print('  Detected version: '+version+'. Required version: '+dipy_version)
     # except ImportError:
     #     print_fail()
     #     install_software = 1
 
     # Check ANTs integrity
     print_line('Check ANTs compatibility with OS ')
     cmd = 'isct_test_ants'
-    # here, cannot use commands.getstatusoutput because status is wrong (because of launcher)
-    # status = os.system(cmd+" &> /dev/null")
-    # status, output = sct.run(cmd, 0)
-    # import subprocess
-    # process = subprocess.Popen(cmd, shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-    # status = subprocess.call(cmd, shell=True)
-    # status = process.returncode
-    (status, output) = commands.getstatusoutput(cmd)
-    # from subprocess import call
-    # status, output = call(cmd)
-    # print status
-    # print output
-    # if status in [0, 256]:
+    status, output = sct.run(cmd, verbose=0, raise_exception=False)
     if status == 0:
         print_ok()
     else:
         print_fail()
-        print output
+        print(output)
         e = 1
     if complete_test:
-        print '>> ' + cmd
-        print (status, output), '\n'
+        print('>> ' + cmd)
+        print((status, output), '\n')
 
     # check if ANTs is compatible with OS
     # print_line('Check ANTs compatibility with OS ')
     # cmd = 'isct_antsRegistration'
-    # status, output = commands.getstatusoutput(cmd)
+    # status, output = sct.run(cmd)
     # if status in [0, 256]:
     #     print_ok()
     # else:
     #     print_fail()
     #     e = 1
     # if complete_test:
-    #     print '>> '+cmd
-    #     print (status, output), '\n'
+    #     print('>> '+cmd)
+    #     print((status, output), '\n')
 
     # check PropSeg compatibility with OS
     print_line('Check PropSeg compatibility with OS ')
-    (status, output) = commands.getstatusoutput('isct_propseg')
-    if status in [0, 256]:
+    status, output = sct.run('isct_propseg', verbose=0, raise_exception=False)
+    if status in (1, 257):
```

Comment:
wouldn't 1 be "not ok"?

Reply:
Turns out that running `isct_propseg`  without arguments returns 1, and this is the expected correct return value...

Reply:
strange... how come this test was fine so far with "0" as "OK"?

Reply:
```python
>>> status, output = commands.getstatusoutput('isct_propseg')
>>> status
256
```
```sh
$ isct_propseg >/dev/null 2>&1; echo $?
1
```

I didn't know `commands.getstatusoutput()`, by checking the docs now I understand, its `status` is *not* the program error code but something containing a bitfield.
```c
#define WEXITSTATUS(status)      ((_W_I(status) >> 8) & 0xFF)
```
In doing a transcription of the code I made a mistake in expecting that `257` (didn't really try to understand, and I just relied on `isct_propseg` return code), I should have put `(0,1)` to be more accurate. I'll fix this, in case the program is modified.